### DISCUSSION TITLE
How to make phone country in filament view and edit page in the form

### DISCUSSION THREAD

**Discussion by sezohessen** [ysfkaya/filament-phone-input#46] [Q&A] [answered]
How to make phone country in filament view and edit page in the form ? 
so that I can see the flag of the country
```
                    PhoneInput::make('Mobile')
                        ->initialCountry()//Should accept record
                        ->inputNumberFormat(PhoneInputNumberType::NATIONAL)
                        ->required(),
```

**Comment by ysfkaya** [comment]
Hello! @sezohessen 

I don't quite understand.

Do you want to show the country state value on an input?

**Reply by sezohessen** [reply]
I `initialCountry` method does not accept Clousre its only accepting string

**Reply by ysfkaya** [reply, marked as answer]
> I `initialCountry` method does not accept Clousre its only accepting string

Try the `version 3`
